README.md

```diff
@@ -18,11 +18,11 @@ You'll need at least Node 12. We use [yarn](https://yarnpkg.com/) for our depend
 
 ## Development mode
 
-1. Clone this repo (e.g. `git clone https://github.com/codimd/react-client.git hedgedoc-react-client`)
+1. Clone this repo (e.g. `git clone https://github.com/hedgedoc/react-client.git hedgedoc-react-client`)
 2. Go inside the repo (e.g. `cd hedgedoc-react-client`)
 3. Run `yarn install`
 4. Either run
-    - `yarn start:dev` (expects [a server](https://github.com/codimd/server/tree/develop) running under [http://localhost:3000](http://localhost:3000))
+    - `yarn start:dev` (expects [a server](https://github.com/hedgedoc/server/tree/develop) running under [http://localhost:3000](http://localhost:3000))
```

Comment:
It's `hedgedoc/hedgedoc` and not `server` anymore.

Reply:
don